ui/src/auth/Login.js

```diff
@@ -9,6 +9,10 @@ import SplashPage from 'shared/components/SplashPage'
 const VERSION = process.env.npm_package_version
 
 const Login = ({authData: {auth}}) => {
+  if (auth.links && auth.links.length===1 && auth.links[0].redirectLogin) {
```

Comment:
```suggestion
  if (Array.isArray(auth.links) && auth.links.length === 1 && auth.links[0].redirectLogin) {
```

this makes it a little more explicit and safe that you're checking for an array - auth.links could exist but be not an array (unlikely, but explicitness is usually good). also adds whitespace around the `===` operator.